app/controllers/api/v1/reports_controller.rb

```diff
@@ -36,15 +37,19 @@ def destroy
 
       def last
         conditions = { :host_id => Host.authorized(:view_hosts).find(params[:host_id]).try(:id) } if params[:host_id].present?
-        max_id = resource_scope.where(conditions).maximum(:id)
+        max_id = resource_scope.my_reports.where(conditions).maximum(:id)
         @report = resource_scope.includes(:logs => [:message, :source]).find(max_id)
         render :show
       end
 
       private
 
+      def deprecated
+        Foreman::Deprecation.api_deprecation_warning("Reports were renamed to ConfigReports")
+      end
+
       def resource_scope(options = {})
-        super(options).my_reports
+        ConfigReport.authorized(:view_config_reports)
```

Comment:
Can you still call super(options) here?  It looks like you can override the controller name.